content/admin/github-actions/enabling-github-actions-for-github-enterprise-server/enabling-github-actions-with-minio-storage.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Enabling GitHub Actions with MinIO Gateway for NAS storage
```

Comment:
@juliandunn would it make sense to keep the existing doc for NAS gateway and add a new doc for minio storage? Technically NAS gateway will keep working if user choses to use their LTS support. We can probably update the warning to cross link and inform user about possibility to using minio storage.

Reply:
I wouldn't object to adding a row in a compatibility table or something, but I don't think we need a whole article about it, particularly if what we actually want is customers to not be running something that's not trending-towards-EOL